pandas/core/frame.py

```diff
@@ -8134,7 +8134,7 @@ def corr(self, method="pearson", min_periods=1) -> "DataFrame":
 
         return self._constructor(correl, index=idx, columns=cols)
 
-    def cov(self, min_periods=None, ddof=1) -> "DataFrame":
+    def cov(self, min_periods: int = None, ddof: int = 1) -> "DataFrame":
```

Comment:
Since these can be None I think we want Optional[int] (same for Series)

Reply:
Done